### ISSUE TITLE
[Brave Browser] Navigation not showing when clicking the profile icon

### ISSUE THREAD

**Issue by DragonDev1906** [Bootstrap-Academy/Bootstrap-Academy#47]
### Bootstrap Academy username

DragonDev1906

### What browsers are you seeing the problem on?

Other (please specify below)

### What happened?

Browser: Brave Version 1.58.137 Chromium: 117.0.5938.153 (Official Build) (64-bit)

When clicking the profile icon (top right, blue/white circle) the navigation dropdown is not visible. The same happens on smaller screens with the burger menu.
Note that there are other things that don't work correctly or differ between the two browsers:
- Firefox shows a circle with an Education hat in the bottom right, Brave doesn't.

# Actual behavior
Firefox: Shows dropdown with 6 entries on click
Brave: Nothing visible happens but a new element is created in the DOM (see screenshots)

# Expected behavior
Both browsers should display the dropdown, without it navigating the site becomes difficult.

# Screenshots
Left: Brave
Right: Firefox

Before clicking:
![image](https://github.com/Bootstrap-Academy/Bootstrap-Academy/assets/8270201/a6083a93-71c9-4f3b-8ba3-51cf0de035c3)

After clicking (when the navigation should be open):
![image](https://github.com/Bootstrap-Academy/Bootstrap-Academy/assets/8270201/5cac0ea6-f24d-4e06-a4fa-c9ea73790277)
![image](https://github.com/Bootstrap-Academy/Bootstrap-Academy/assets/8270201/44296fa8-2b46-4ef8-8e50-885f86a19c01)

**Comment by Onako2** [comment]
It could be that the content blocker in Brave is blocking it(Didn't try yet). Try to turn the content blocker off for the website. I experienced the same thing while using u-Block Origin

**Comment by DragonDev1906** [comment]
Good point, it looks like Brave detects it as `tracker` or `ad`, it works if I disable the first option. Though for some reason it doesn't show that anything was blocked, so I'm not sure why Brave thinks it should be blocked.
![image](https://github.com/Bootstrap-Academy/Bootstrap-Academy/assets/8270201/040740aa-c12a-43f9-b364-6d75c76ca679)

**Comment by Onako2** [comment]
I think they are using the same filter list. We should report to them that it breaks the site. In uBlock Origin there is an official way to report it
